docs/AboutUs.adoc

```diff
@@ -4,18 +4,18 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
+README was developed by the https://github.com/CS2103-AY1819S2-W10-1[CS2103-AY1819S2-W10-1] team. +
```

Comment:
Probably should mention that we built off the original team's AB4

Reply:
Fixed in 35c2ae0